app/src/main/java/org/mozilla/fenix/share/ShareViewModel.kt

```diff
@@ -89,12 +94,32 @@ class ShareViewModel(application: Application) : AndroidViewModel(application) {
                 type = "text/plain"
                 flags = Intent.FLAG_ACTIVITY_NEW_TASK
             }
-            val shareAppsActivities = getIntentActivities(shareIntent, getApplication())
-            var apps = buildAppsList(shareAppsActivities, getApplication())
+            val shareAppsActivities = getIntentActivities(shareIntent, context)
+
+            var apps = buildAppsList(shareAppsActivities, context)
             recentAppsStorage.updateDatabaseWithNewApps(apps.map { app -> app.activityName })
             val recentApps = buildRecentAppsList(apps)
             apps = filterOutRecentApps(apps, recentApps)
 
+            val tint = ThemeManager.resolveAttribute(R.attr.secondaryText, context)
```

Comment:
just want to make sure this is the correct color. Do the icons use the same coloring as text in the share sheet?

Reply:
There's no other example of what colour to use here.  All other icons are retrieved from package manager (launcher icons)  I found that primary text colour was too bright and this is just right.

Reply:
cc: @gabrielluong

Reply:
Let's send a screenshot of this over to Crystal and get a confirmation.

Reply:
Sure, please see attached screenshot.  I've also contacted Crystal.